src/hooks/screen/useRootChannelListHook.ts

```diff
@@ -29,6 +40,76 @@ const useRootChannelListHook = () => {
     }
   };
 
+  const requestPermission = async () => {
+    const authStatus = await messaging().requestPermission();
+    const enabled =
+      authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+      authStatus === messaging.AuthorizationStatus.PROVISIONAL;
+
+    if (enabled) {
+      const fcmToken = await messaging().getToken();
+      const payload = {
+        fcm_token: fcmToken
+      };
+      fcmTokenService(payload);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `requestPermission` function is defined but never used within the provided code. If it's not used elsewhere in the file, consider removing it to reduce complexity and improve maintainability.
```diff
- const requestPermission = async () => {
-   const authStatus = await messaging().requestPermission();
-   const enabled =
-     authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
-     authStatus === messaging.AuthorizationStatus.PROVISIONAL;
- 
-   if (enabled) {
-     const fcmToken = await messaging().getToken();
-     const payload = {
-       fcm_token: fcmToken
-     };
-     fcmTokenService(payload);
-   }
- };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->